api.planx.uk/send/uniform.js

```diff
@@ -35,6 +35,16 @@ const sendToUniform = async (req, res, next) => {
     });
   }
 
+  // confirm that this session has not already been successfully submitted before proceeding
+  const submittedApp = await checkUniformAuditTable(payload?.sessionId);
+  if (submittedApp?.submissionStatus === "PENDING" && submittedApp?.canDownload) {
```

Comment:
it's trickier to check the uniform response for "success", but I think this combination is sufficient? open to other ideas here!

Reply:
Again, it could be worth improving what we capture back from Uniform to have a bit more certainty here - but I think there's zero problems with this and it does describe the check correctly 👍